### PR TITLE
Draft of Karmada partner guideline

### PR DESCRIPTION
**Pull request by wawa0210**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by RainbowMango — COMMENTED** [review]
/assign

File: commercial-support.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Commercial support plan
+
+**Note:** This document is a work in progress
+
+The Karmada commercial service provider program offers Karmada support, consulting, professional services for organizations embarking on their Karmada journey.
+
+The Karmada commercial service provider program ensures that enterprises get the support they’re looking for to roll out new multi-cluster applications more quickly and more efficiently than before, while feeling secure that there’s a trusted and vetted partner that’s available to support their production and operational needs.
```
Comment (RainbowMango) [inline commercial-support.md:7]:
```suggestion
Karmada commercial partners are any organization that successfully leverages the Karmada project for their service offering. Such as:
- Using Karmada to build service on a public or private cloud.
- Repackaging Karmada as a core component of a service offering.
- Offering Karmada technical support, consultation services, and training.
- Others leverage Karmada in the manner it was intended to serve their customers.
```
general introduction/definition of `commercial partner`.

File: commercial-support.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Karmada commercial service provider details
```
Comment (RainbowMango) [inline commercial-support.md:9]:
```suggestion
## Why become a partner?
```

File: commercial-support.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* More community meeting exposure opportunities, such as the following meetings
+  * KubeCon.
+  * Online and offline meetings held by the Karmada community.
+  * Online and offline meetings participated by the Karmada community(such as KCD).
+* CNCF official media exposure opportunities, such as official Blog, Twitter, public accounts, etc.
+* Karmada security vulnerability disclosed in advance.
+* Karmada Community Supported Job Postings.
+* Community business opportunity delivery and connection.
+
+### Karmada commercial service provider process
```
Comment (RainbowMango) [inline commercial-support.md:24]:
```suggestion
## How to become a partner?
```

**Review by RainbowMango — COMMENTED** [review]
Review is still on going.

File: commercial-support.md (hunk 3 above)
Comment (wawa0210) [reply commercial-support.md]: done

File: commercial-support.md (hunk 2 above)
Comment (wawa0210) [reply commercial-support.md]: done

File: commercial-support.md
[the remark this answers is not in the document]
Comment (wawa0210) [reply commercial-support.md:1]: Sounds good

File: commercial-partner.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+### How to become a partner?
+
+If your company is interested in becoming a Karmada commercial partner, please complete the following steps:
+
+1. Prepare the following information in advance
+   * Company official name.
+   * Company official website address(related to the Karmada commercial partner).
+   * Company Contact information email or github ID.
+   * Company logo (png format).
+2. In [community](https://github.com/karmada-io/community.git) repo, select the issue template of `Karmada support Partner Request` to create an issue and fill in the above information.
```
Comment (RainbowMango) [inline commercial-partner.md:37]:
```suggestion
2. [Open an issue](https://github.com/karmada-io/community/issues/new?assignees=&labels=&projects=&template=commercial-support-partner.yml&title=REQUEST%3A+Organization+%3COrganization+name%3E+that+provides+commercial+support+to+Karmada+New+Commercial+Support+) against the karmada-io/community repo and fill in the relevant information.
```

File: commercial-partner.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- Repackaging Karmada as a core component of a service offering.
+- Offering Karmada technical support, consultation services, and training.
+- Others leverage Karmada in the manner it was intended to serve their customers.
+
+### Why become a partner?
+
+If your company provides professional services to support Karmada deployments, please consider applying to become a Karmada commercial partner.
+
+The main benefits of becoming a Karmada commercial partner are:
+* Placement on the first tab of https://karmada.io/
```
Comment (kevin-wangzefeng) [inline commercial-partner.md:18]: It's not clear to me, where is the first tab?

File: commercial-partner.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+* More community meeting exposure opportunities, such as the following meetings
+  * KubeCon.
+  * Online and offline meetings held by the Karmada community.
+  * Online and offline meetings participated by the Karmada community(such as KCD).
+* CNCF official media exposure opportunities, such as official Blog, Twitter, public accounts, etc.
```
Comment (kevin-wangzefeng) [inline commercial-partner.md:23]: The description here may be a bit confusing, as you know the community should stay open and neutral, we should not use community channels to promote any commercial content, e.g. product pitch from any service provider.

Thus highlighting content neutrality would be helpful.
You may refer to "Marketing services and programs" section at https://www.cncf.io/services-for-projects/#marketing

File: commercial-partner.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### How to become a partner?
+
+If your company is interested in becoming a Karmada commercial partner, please complete the following steps:
+
+1. Prepare the following information in advance
+   * Company official name.
+   * Company official website address(related to the Karmada commercial partner).
+   * Company Contact information email or github ID.
+   * Company logo (png format).
```
Comment (kevin-wangzefeng) [inline commercial-partner.md:36]: Svg format is also an acceptable option

File: commercial-partner.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+2. In [community](https://github.com/karmada-io/community.git) repo, select the issue template of `Karmada support Partner Request` to create an issue and fill in the above information.
+
+Karmada maintainers will review these issues, and if the information is correct, a formal PR will be created. At this time, the applicant organization will officially join the Karmada Commercial Partner.
+
+### Karmada commercial partner exit mechanism
+
+There is currently no withdrawal mechanism, but the Karmada Community Governance Committee reserves the right to withdraw commercial partners.
```
Comment (kevin-wangzefeng) [inline commercial-partner.md:43]: We don't have "Karmada Community Governance Committee" yet, so maybe keep Karmada maintainer team here.

**Review by kevin-wangzefeng — COMMENTED** [review]
Thanks for initiating this. The program is very imporntant to encourage more organizations join the communtiy explicitly.

Other than a few comments about phrasing aspect, I don't see big issue.

File: commercial-partner.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+# Commercial Partner
```
[the remark this answers is not in the document]
Comment (RainbowMango) [reply commercial-partner.md:1]: That makes sense, no commercial business between the community and partners :)

So, how about `Community Partner`?

File: commercial-partner.md (hunk 5 above)
Comment (RainbowMango) [reply commercial-partner.md:18]: nice finding. It should be the first tab of `https://karmada.io/partners`.

By the way, the page will be ready after https://github.com/karmada-io/website/pull/531, but the URL should be right.

File: commercial-partner.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (wawa0210) [reply commercial-partner.md:1]: Maybe we can refer to KCSP(KUBERNETES CERTIFIED SERVICE PROVIDER)

But how does the community define this certification? We are currently open and welcome more partners to join.

File: commercial-partner.md (hunk 5 above)
Comment (wawa0210) [reply commercial-partner.md:18]: 👍

File: commercial-partner.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (RainbowMango) [reply commercial-partner.md:1]: `Community Partner` could upgrade to `certified service provider` after we set up the certification process in the future.

[the remark this answers is not in the document]
Comment (wawa0210) [reply commercial-partner.md]: done

File: commercial-partner.md (hunk 8 above)
Comment (wawa0210) [reply commercial-partner.md]: done

**Comment by RainbowMango** [comment]
/lgtm
/approve

Thanks.
Note that, the document is a work in progress, we might iterate it at any time.

**Pull request merged**